### PR TITLE
[IMP] survey: improve survey fill finished page

### PR DESCRIPTION
**Pull request by lm4649**
This commit makes some visual adjustments to the survey completion page:

1. Remove "Thank you" label when there is a scoring enabled and display "You scored X.XX %"

2. Remove the border on the survey navigation buttons at the bottom right of the page

task-3918542

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/survey/views/survey_templates.xml [hunk 1]
```diff
@@ -283,12 +283,16 @@
     <template id="survey_fill_form_done" name="Survey: finished">
         <div class="wrap">
             <div class="o_survey_finished mt32 mb32">
-                <h1 class="fs-2">Thank you!</h1>
+                <h1 class="fs-2">
+                    <t t-if="survey.scoring_type != 'no_scoring'">
```
Comment (Odoonan) [inline addons/survey/views/survey_templates.xml:287]: Hello @lm4649 ,

Just a question here, I see we will display the score even if we do not have a scoring_success_min, which was not the case before (see condition below). Do we want to change this? I was not sure reading the pad.

You can green if the condition is fine :) Cheers!

Comment (lm4649) [reply addons/survey/views/survey_templates.xml:287]: Hello @Odoonan,

Thanks for checking this PR :pray: 
I guess it's want we want, see [https://github.com/odoo/odoo/pull/165032](url)

File: addons/survey/views/survey_templates.xml [hunk 2]
```diff
@@ -283,12 +283,16 @@
...
+                        You scored <t t-out="answer.scoring_percentage" />%
```
Comment (awa-odoo) [inline addons/survey/views/survey_templates.xml:288]: While we're at it, can we remove the space before the tag closing?

**Review by awa-odoo — COMMENTED** [review]
Hello @lm4649 

A very important comment in the review (hum).

Additionally, it looks like you have a file conflict to resolve.

Thanks & cheers!

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**